### PR TITLE
Fix typo on toolbar

### PR DESCRIPTION
**Pull request by jeongwhanchoi**
I found the typo on translation.json file like [below commit](https://github.com/dbis-uibk/relax/pull/16/commits/3b78cfbe4b5ffc51336b34581782204d6b14df02)

### CODE REVIEW

**Review by laxman954 — COMMENTED** [review]
Changes looks good, this is just typo changes

**Review by epsimatic — APPROVED** [review]
Simple typo fixes, nothing special

**Comment by shauniop** [comment]
Simple typo fix, doesn't need any more reviews to merge.

**Review by Paulophmp — APPROVED** [review]
Ok!

File: locales/en/translation.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
 				"warning-distinct-missing": "DISTINCT is missing; relational algebra uses implicit duplicate elimination",
 				"warning-ignored-all-on-set-operators": "ignored ALL on set operation; relational algebra uses implicit duplicate elimination",
 				"error-variable-name-conflict": "name conflict: relation name \"__name__\" already exists",
-				"error-variable-cyclic-usage": "cylic usage of variable \"__name__\" detected"
```
Comment (rashmiv01) [inline locales/en/translation.json:52]: spelling correction required, should be mentioned as cyclic usage of variable, but its cylic in line number 52

**Review by NeuroWinter — APPROVED** [review]

**Comment by KunalKK99** [comment]
it's working

**Pull request merged**